### ISSUE TITLE
"Effective Damage Increase" for Scholar

### ISSUE THREAD

**Issue by towelcat** [baaron4/GW2-Elite-Insights-Parser#72]
Noticed a 'bug' in the parser with it.

https://i.imgur.com/oUd1hLR.png

Displayed increase is calculated via (scholar bonus) / (total damage), which isn't very useful when you're trying to see how much it increased the physical damage output.

Instead, try:

```
Math.Round(100 * (dps.bossPowerDamage / (dps.bossPowerDamage - stats.scholarDmg - 1))
```
(or whatever it would be)

As an example of why the current method is wrong: consider you want to see how much bonus damage you're getting out of a force sigil (5%). Let's say you're doing 100 physical damage base (105 with force), and 20 condi damage.

Current method would display 5 / 125 = 4% increase, kinda useless?

Change to 100 physical (105 with sigil) and 0 condi.

5 / 105 = 4.76%, still wrong.

105 / (105 - 5) - 1 = 5%, correct number

**Comment by EliphasNUIT** [comment]
Yeah you are right what we are showing today is the % of scholar damage and not the increase. I'll correct it.

However what's the reason for wanting to remove condi from that? Is it more interesting to know how much you buffed your physical damage than your total dps?

**Comment by EliphasNUIT** [comment]
Seems reasonable, I'll change it. Thanks for the feedback ^^

**Comment by EliphasNUIT** [comment]
Fixed in 1.5.3.1
